backend/hct_mis_api/apps/household/forms.py

```diff
@@ -220,11 +220,16 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:
             queryset=Program.objects.filter(business_area_id=self.business_area_id, status=Program.ACTIVE),
             label="Select a program to enroll households to",
         )
+        self.fields["rdi"] = forms.ModelChoiceField(
+            queryset=RegistrationDataImport.objects.filter(business_area_id=self.business_area_id),
+            label="Select a RDI to enroll households to",
+        )
```

Comment:
I don't think you should select RDI - I think we want to just create a new RDI for the enrolled records now. But I think @MarekBiczysko was already working on that?

Reply:
I don't think it would be a good approach to add records to already existing RDI. I think we would need a new RDI

Reply:
updated